ideascube/library/views.py

```diff
@@ -184,3 +184,34 @@ def get_row(self, book):
         return row
 
 book_export = staff_member_required(BookExport.as_view())
+
+
+class BookSpecimenExport(ZippedCSVExportMixin, View):
+
+    fields = ['item', 'barcode', 'serial', 'comments', 'location', 'file']
+    model = BookSpecimen
+    prefix = 'book_specimens'
+
+    def get_headers(self):
+        return self.fields
+
+    def get_row(self, bookspecimen):
+        row = {}
+
+        for field in self.fields:
+            if field == 'item':
+                row[field] = bookspecimen.item.name
```

Comment:
Having just "name" to match the book may generate some issues, I'm tempted to add also the book ISBN (that may be null) to reduce the problematic cases.